src/components/Countries/Countries.test.js

```diff
@@ -0,0 +1,41 @@
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import store from '../../Redux/createStore';
+import Country from './Country/Country';
+import Countries from './Countries';
+
+describe('Countries component test', () => {
+  test('Countries snapshot', () => {
+    const props = {
+      name: 'country 1',
+      image: 'country 1 image',
+      total: 100,
+      newCases: 50,
+      pageUrl: 'url',
+    };
+    const {
+      name, image, total, newCases, pageUrl,
+    } = props;
+    render(
+      <Router>
+        <Provider store={store}>
+          <Country
+            name={name}
+            image={image}
+            total={total}
+            newCases={newCases}
+            pageUrl={pageUrl}
+          />
+        </Provider>
+      </Router>,
+    );
+    expect(render(
+      <Router>
+        <Provider store={store}>
+          <Countries />
+        </Provider>
+      </Router>,
+    )).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] (Required) Great work using _**Jest**_ snapshots to test your component. Remember that one of the major requirements for this project is to use _**React Testing Library**_ to carry out integration tests for all your React components. This is a great [tutorial](https://www.freecodecamp.org/news/react-testing-library-tutorial-javascript-example-code/) that shows you how to use _**React Testing Library**_ to test your components. Kindly implement this change. 👍